README.md

```diff
@@ -348,6 +349,7 @@ You can integrate a form (such as a newsletter subscription form) with Metrilo u
     email: 'newsletter[fields][1]', // required - the name attribute of the "email" input field
     firstName: 'newsletter[fields][0][first]', // optional - the name attribute of the "first name" input field
     lastName: 'newsletter[fields][0][last]', // optional - the name attribute of the "last name" input field
+    phoneNumber: 'newsletter[fields][0][first]', // optional - the name attribute of the "phone number" field
```

Comment:
Make this one a different value than the one for firstName - like `newsletter[fields][3]`

Reply:
done, pls check it out again